docs/deployment/caching/index.md

```diff
@@ -0,0 +1,41 @@
+---
+myst:
+  html_meta:
+    "description": "Caching Plone"
+    "property=og:description": "Caching Plone"
+    "property=og:title": "Caching Plone"
+    "keywords": "Plone, deployment, automation, caching"
+---
+
+(caching-label)=
+
+# Caching
+
+## Overview
+
+HTTP caching is a technique used to speed up the delivery of web content by storing previously requested resources (such as images, scripts, and stylesheets) in a cache. When a user requests a resource, the cache can serve the resource directly from its storage, rather than having to fetch it from the original source.
+
+It is possible to use a web accelerator, like {term}`Varnish` or {term}`Squid`, to implement HTTP caching in your own premises. The usual setup will have the web accelerator installed between Plone and the Internet, and when a request is made to the site the web accelerator will intercept the request and check to see if it has a cached copy of the requested resource. If a cached copy is found, the accelerator will serve the cached copy directly to the client, otherwise it will make a request to the backend Plone server and then store a copy on the content in its local cache.
```

Comment:
Do we really want to mention Squid these days? 
Also here: split to one sentence per line
```suggestion
It is possible to use a web accelerator, like {term}`Varnish` to implement HTTP caching in your own premises. 
The usual setup will have the web accelerator installed between Plone and the Internet, and when a request is made to the site the web accelerator will intercept the request and check to see if it has a cached copy of the requested resource. 
If a cached copy is found, the accelerator will serve the cached copy directly to the client, otherwise it will make a request to the backend Plone server and then store a copy on the content in its local cache.
```